docs/getting-started/quickstart-install.md

```diff
@@ -0,0 +1,107 @@
+# Install Knative using Quickstart
+
+This topic describes how to install Knative Serving and Eventing using
+the Knative Quickstart plugin.
+The plugin installs a preconfigured Knative deployment on a local Kubernetes cluster.
+
+!!! warning
+    Knative Quickstart Environments are for experimentation use only. For a production ready installation, see [Installing Knative](../install/README.md).
+
+## Before you begin
+
+Before you can get started with a Knative Quickstart deployment you must install kind, the Kubernetes CLI, and the Knative CLI.
+
+### Prepare a local Kubernetes cluster
+
+You can use [`kind`](https://kind.sigs.k8s.io/docs/user/quick-start){target=_blank} (Kubernetes in Docker) or [`minikube`](https://minikube.sigs.k8s.io/docs/start/){target=_blank} to run a local Kubernetes cluster with Docker container nodes.
```

Comment:
Since we're creating the cluster with quickstart, we can probably drop these lines (maybe add the links to the sentence above?)

Reply:
Or maybe change the heading to "Install local cluster tool" or something along those lines (but better :) ) -- the heading as it is sounds like I'm going to set up the cluster, rather than just needing to have one of those binaries on my machine

Reply:
Good point. I'll see if I can improve this.